webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require ('webpack')
+
+module.exports = {
+    entry: './source/likely.js',
+    output: {
+        filename: 'likely.js',
+        library: 'likely',
```

Comment:
@iamakulov Seems, in that case, it breaks [the current behaviour](https://github.com/ilyabirman/Likely/blob/master/source/likely.js#L5). It should be precisely `likely`.